src/convertToObject.js

```diff
@@ -1,18 +1,28 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
```

Comment:
![image](https://github.com/mate-academy/js_style-to-object/assets/96409563/86fec238-4a28-4635-9004-da2eebcb1a51)
Let's specify what exactly the result is